metricbeat/module/logstash/node/node.go

```diff
@@ -65,11 +65,11 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 // It returns the event which is then forward to the output. In case of an error, a
 // descriptive error must be returned.
 func (m *MetricSet) Fetch() (common.MapStr, error) {
-	data, err := m.http.FetchJSON()
+	content, err := m.http.FetchContent()
```

Comment:
Is there an advantage in unsing FetchContent? Or should we rather use FetchJson everywhere?

Reply:
I think it would be better to use `FetchJSON` everywhere. However that would mean updating all tests files that read from the test fixture JSON files to unmarshall the JSON before passing it to the event parsing function. For example, over here:

https://github.com/elastic/beats/pull/8497/files#diff-ae4c91f590d2739e37b1deb62691f913R32

So its six of one, half a dozen of the other. Which do you prefer?

Reply:
I think I also prefer `FetchJSON`. ++ on a follow up PR.